lab-06-lindsay/test/test.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const assert = require('assert');
+const app = require('../server.js');
+const expect = require('chai').expect;
+
```

Comment:
Try writing a before/after to setup your server before your run your tests. Also, write tests for each of your commands for full credit!